src/components/calcite-dropdown-item/calcite-dropdown-item.tsx

```diff
@@ -108,6 +108,11 @@ export class CalciteDropdownItem {
         scale={iconScale}
       />
     );
+    const contentNode = (
```

Comment:
I don't totally get the need for this, couldn't the margin auto just be applied as it was before?

Reply:
I can look at this again.

Reply:
Not a huge deal just wondering if there was some weird @apply limitation that made it hard to work with. If it works like this it's probably just fine.

Reply:
Oh yeah...this was because there still needs to be some margin between the text and the icon. So one node needs the auto margin to achieve the layout. And the other needs the set margin (e.g. 0.75rem) to create that space. It made more sense to have that set margin value only when it is relevant, i.e. when that end icon exists.

Reply:
Cool, makes sense, thanks!